### ISSUE TITLE
Question regarding moving the package to aws lambda layer

### ISSUE THREAD

**Issue by georgetakla** [psycopg/psycopg#727]
Thanks for maintaining this lib @dvarrazzo !!
The package works well when I installed the package within an AWS lambda and imported it into the app. When I move the package to  a lambda layer (the same requirement.txt is moved to another folder referenced by the function as a layer) I get the following errors when I use sam local invoke to start the function on docker;

> ERROR] Runtime.ImportModuleError: Unable to import module 'app': no pq wrapper available.
> Attempts made:
> - couldn't import psycopg 'c' implementation: No module named 'psycopg_c'
> - couldn't import psycopg 'binary' implementation: cannot import name 'pq' from 'psycopg_binary' (/opt/python/psycopg_binary/__init__.py)
> - couldn't import psycopg 'python' implementation: libpq library not found

I appreciate your thoughts on this

**Comment by dvarrazzo** [comment]
It seems that whatever you have put in `/opt/python/psycopg_binary/` is not correct. Python cannot import it. I don't know anything more than that.

I am unable to do any work for free regarding AWS. I can only look into it as a paid consultancy. I am sure you understand.

Closing because this is not a psycopg issue.

**Comment by georgetakla** [comment]
I understand. I solved the issue and sharing 
I am using mac with M1 chipset, and I specified in the template file for lambda to use arm64 

> Properties: 
>   ...
>   Architectures:
>     -arm64

So, the pyscopg[binary] installation worked.

In the lambda layer, without specifying the architecture, the imported binary package is for mac M1 and does not work. so, I added the architect to the layer metadata as explained [here](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/building-layers.html) and it worked!!

>     Metadata:
>       BuildMethod: python3.11
>       BuildArchitecture: arm64
